### DISCUSSION TITLE
2022-06-25-mac

### DISCUSSION THREAD

**Discussion by tw93** [tw93/tw93.github.io#104] [General]
https://tw93.fun/2022-06-25/mac.html 

 Hi，我是 Tw93，是一个前端工程师，空闲也折腾 iOS / macOS ，喜欢开源分享交流。Tool何为差生文具多，看完这个你就清楚了， 用于2022年6月16日团队双周会的一个快闪分享。

**Comment by tw93** [comment]
测试一下新的评论系统

**Comment by xcuYao** [comment]
test2

**Comment by malinkang** [comment]
test3

**Comment by Lantianyou** [comment]
WeRead 是用 Tauri 写的吗？

**Comment by tw93** [comment]
> @Lantianyou
> WeRead 是用 Tauri 写的吗？
是的

**Comment by cwandev** [comment]
PrettyClean链接过去的地址下载不能用了，

使用这个: https://www.prettyclean.cc/zh

**Comment by tw93** [comment]
> @CharleeWa
> PrettyClean链接过去的地址下载不能用了，
> 
> 使用这个: https://www.prettyclean.cc/zh

谢谢反馈，已经替换

**Comment by jincaiw** [comment]
谢谢分享

**Reply by whsheng** [reply]
**感谢作者用心推荐！，清理软件，个人推荐腾讯的「柠檬清理，Lemon」**

**Reply by tw93** [reply]
柠檬最近有人反馈出了一些小问题、删除错了东西，注意使用的时候小心一丢丢哦

**Comment by windystreet739** [comment]
感谢作者分享🐂

**Comment by jincaiw** [comment]
感觉装的软件太多，mac 16G的内存优点吃力啊

**Reply by tw93** [reply]
其实我的软件算很少的啦，哈哈哈哈，不过今年更少了，有空分享分享

**Comment by lorrylockie** [comment]
11、14、16、19 可以用一个软件替换 raycast

**Comment by duofu** [comment]
> ‘App Cleaner & Uninstaller’ 扫描得比较完整，‘MacClean Pro’ 套件之一
App Cleaner/PrettyClean + mac-cleanup-sh，那为什么不直接上 ‘MacClean Pro’ 或者 ‘CleanMyMac X’(有破解版的) 呢？

> ‘Shottr’ 功能比 ‘Xnip’ 要强，但识别性能（尤其大面积空白时）比‘Picsee’要弱

> ‘Hidden Bar’ 体验上不如 ‘Bartender’(有破解版的) 

> ‘MindNode’ 和 ‘XMind’ 还是倾向于后者，尤其2023版本升级后，‘XMind’性能提高了很多（但不能自定义背景图是一大槽点）

> ‘Leaf’ 和 ‘Feeder’ 差不多，功能上感觉不如 ‘NetNewsWire’

**Comment by pingmoogle** [comment]
蟹蟹作者，zsh-autosuggestions很好用😋
